lifelines/tests/test_estimation.py

```diff
@@ -330,10 +331,16 @@ def test_print_summary(self, sample_lifetimes, known_parametric_univariate_fitte
             f.print_summary(style="latex")
 
 
+class ExponentialBaseMixtureCureFitter(MixtureCureFitter):
+    def __init__(self):
+        super().__init__(base_fitter=ExponentialFitter())
+
+
 class TestUnivariateFitters:
     @pytest.fixture
     def univariate_fitters(self):
         return [
+            ExponentialBaseMixtureCureFitter,
```

Comment:
this is fine, but can't you also do:

```
return 
  [ 
    MixtureCureFitter(ExponentialFitter()),
   ...
```

Reply:
The univariate_fitters fixture returns a list of callables, rather than class instances, so that wouldn't work directly. You see this in the later tests which need more than once instance of the class, and do things like:
```
        for fitter in univariate_fitters:
            f1 = fitter()
            f2 = fitter()
```
However, your suggestion made me realize that a simple lambda might actually be less weird than that extra class I had. So I've committed an example of what that might look like for you. It also has a side effect of fixing two tests that I broke :)